src/x_config_gen.rs

```diff
@@ -0,0 +1,3 @@
+#[macro_use]
```

Comment:
this is obsolete, you should annotate your `macro_rules` with `#[macro_export]` instead